pages/checkout.vue

```diff
@@ -409,18 +369,130 @@ export default {
             },
           },
           isDestinationCommercial: false,
-          shippingMethodCode: "",
-          shippingMethodName: shippingDetails.value.shippingMethod,
+          shippingMethodCode: shippingDetails.value.shippingMethod.code,
+          shippingMethodName: shippingDetails.value.shippingMethod.name,
         },
       }
 
       await setShippingInfo(params)
+      populateShippingDetails()
+
+      await loadShippingMethods(checkout.value.id)
+      populateShppingMethodDetails()
+    }
+
+    // shippingMethods
+    const shippingMethodDetails = ref({
```

Comment:
these should just be computed properties

`const shipItems = computed(() => checkoutGetters.getShipItems(checkout.value))`

Reply:
Resolved